Topics/User_Experience/Design_Requirements.md

```diff
@@ -0,0 +1,120 @@
+# **Design Requirements**
+
+# Table of Contents
+
+### [Introduction](#introduction)
+
+### [Motivation](#motivation)
+
+### [Functional Design Requirements](#functional-design-requirements)
+
+### [Non-Functional Design Requirements](#non-functional-design-requirements)
+
+### [Extracting Action Items](#extracting-action-items)
+
+### [References](#references)
+
+### [Additional Resources](#additional-resources)
+
+## **Introduction**
+
+Design requirements are expressions which outline “what” the desired product should achieve and “how” it should achieve them, corresponding to the usefulness and usability of designs.
+
+The usefulness of a product is addressed by the question “Are we designing the right thing?” [1], which considers whether this product is needed by users. 
+
+The usability of a product is addressed by the question “Are we designing the thing right?” [1], which considers whether users are able to use the product.
+
+To achieve this goal, design requirements include specifications related to user interfaces, navigation, visual design, and additional elements which contribute to the satisfaction of end users and overall usability. This makes design requirements an integral part of the user experience.
+
+## **Motivation**
+
+When working with partner clients, scoping the problem that clients want to address and understanding the requirements that their ideal solution entails is very important. Establishing design requirements is highly valuable for CSC301 students, who need to create minimum-viable-products (MVPs) from scratch.
+
+### **1. Client Needs** 
```

Comment:
I think it may be nice to include even more information on how design requirements address client needs here. Maybe briefly mentioning the motivation for **clients** to partake in extracting design requirements.

Reply:
Thank you for this suggestion, I agree that understanding the motivation of clients might help students better format their meetings with clients even. I will add more information about this.

Reply:
That sounds great!